runelite-client/src/main/java/net/runelite/client/ui/overlay/worldmap/WorldMapOverlay.java

```diff
@@ -168,9 +173,44 @@ public Dimension render(Graphics2D graphics)
 			drawTooltip(graphics, tooltipPoint);
 		}
 
+		if (debugInfo)
+		{
+			drawDebugInfo(graphics);
+		}
+
 		return null;
 	}
 
+	private void drawDebugInfo(Graphics graphics)
```

Comment:
There is a whole debug plugin for world map already... can't this be there?

Reply:
Yeah I'll move it over there